new/LIT7048Qene.xml

```diff
@@ -0,0 +1,75 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7048Qene" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ከልአኪ፡ አሮን፡ በበትረ፡ እዴሁ፡ ጥልልት።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Kalʾa-ki ʾAron ba-batra ʾǝde-hu ṭǝllǝlt (Qǝne)</title>
+                <title xml:lang="en" corresp="#t1">Aron's hand prevented you to be fertile in the offspring (?) ...</title>
```

Comment:
Aron prevented you with the verdant stick of his hand ... 
"Stick" can be femenine (ወሠረጸት፡ በትር...), the allusion he is to the stick of Aron that shoot sprouts (Num 17).